routes/index.js

```diff
@@ -35,6 +35,28 @@ const routes = [
     method: 'GET',
     handler: require('../handlers/search'),
   },
+
+  {
+    path: '/problems/{id?}',
```

Comment:
Yes, that was the remnants of a `/trials` route. I fixed it there as well in another pull request :+1: